### PR TITLE
Replace old ODS icons with new ODS icons

### PR DESCRIPTION
**Pull request by sepehr2github**
## Description

Please include a summary of the change and be sure you follow the contributions rules we do provide [here](./CONTRIBUTIONS.md)

Task: [APP-app-2764](https://aragonassociation.atlassian.net/browse/APP-2764)

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] Any dependent changes have been merged and published in downstream modules.
- [x] I ran all tests with success and extended them if necessary.
- [ ] I have updated the `CHANGELOG.md` file in the root folder.
- [x] I have tested my code on the test network.

### CODE REVIEW

File: src/containers/smartContractComposer/components/contractAddressValidation.tsx [hunk 1]
```diff
@@ -565,7 +563,13 @@ const ContractAddressValidation: React.FC<Props> = props => {
                   <Link
                     external
                     type="neutral"
-                    iconRight={<IconLinkExternal height={13} width={13} />}
+                    iconRight={
+                      <Icon
+                        icon={IconType.LINK_EXTERNAL}
+                        height={13}
+                        width={13}
```
Comment (cgero-eth) [inline src/containers/smartContractComposer/components/contractAddressValidation.tsx]: The component does not support the `height` and `width` attributes but 3 different sizes: 
sm => 12px
md => 16px
lg => 20px
For this case we can probably just use `size="sm"`: 
```suggestion
                        size="sm"
```
The same applies to all the other `height` and `width` usage

**Review by cgero-eth — COMMENTED** [review]
Thanks for the changes @sepehr2github!

**Comment by cgero-eth** [comment]
Should we update the components under `@aragon/ods` old as well to use the new icons so that we can remove all the old ODS icons?

File: src/containers/smartContractComposer/components/contractAddressValidation.tsx (hunk 1 above)
Comment (sepehr2github) [reply src/containers/smartContractComposer/components/contractAddressValidation.tsx]: Thanks for pointing out, fixed!

**Review by cgero-eth — COMMENTED** [review]
Let's update the old ODS components as well and remove the old icons as discussed during the daily 👌🏻

**Comment by sepehr2github** [comment]
> Let's update the old ODS components as well and remove the old icons as discussed during the daily 👌🏻

https://discord.com/channels/672466989217873929/1202199749747277824/1202207682979954739

According to the thread, we are good to go!

**Review by cgero-eth — APPROVED** [review]
Thanks @sepehr2github! ✅

**Pull request merged**